### DISCUSSION TITLE
Contrast issue in GitHub workflow badge in dark theme

### DISCUSSION THREAD

**Discussion by humble-barnacle001** [community/community#10121] [Mobile] [answered]
The badges of a GitHub workflow which has been added to README is viewed on the mobile app, the badge is almost unreadable. This issue does not exist on the GitHub website with dark theme.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up @humble-barnacle001 — do you have screenshots or a public repository link that we can use to test and reproduce the issue?

**Reply by humble-barnacle001** [reply]
Public repo link: https://github.com/humble-barnacle001/fetch-covid-update-bot

**Reply by humble-barnacle001** [reply]
GitHub mobile in dark mode:

![IMG_20220122_140022](https://user-images.githubusercontent.com/58736240/150631060-eb479c13-190f-4550-84f3-0444ef4ccbe2.jpg)

Vs

GitHub website in dark mode:
![IMG_20220122_140137](https://user-images.githubusercontent.com/58736240/150631079-1902c375-a454-452c-9437-c7e12aae95ee.jpg)

**Reply by brianlovin** [reply]
Thanks for the context! I'll make sure this gets seen by our engineering team 😊
